python/README.md

```diff
@@ -0,0 +1,141 @@
+# Chericat Python
```

Comment:
`/README.md` should somehow reference this. Otherwise, I'm afraid people won't read it.